packages/use-i18n/src/__tests__/usei18n.test.tsx

```diff
@@ -20,7 +22,8 @@ type NamespaceLocale = {
 
 type OnTranslateError = ComponentProps<typeof I18n>['onTranslateError']
 
-const defaultSupportedLocales = ['en', 'fr', 'es']
+const isLocaleSupportedDefault = (locale: string): locale is Locales =>
```

Comment:
```suggestion
const isDefaultSettingsLocalesSupported = (locale: string): locale is Locales =>
```

sound better ?

Reply:
Agree, I will just remove settings in the name, as we only check for the list of locales and not the whole settings list of locale